e2e_example/pubspec.yaml

```diff
@@ -1,23 +1,28 @@
 name: e2e_example
+description: A web app that uses AngularDart Components
```

Comment:
Looks like it doesn't use the components yet? I think we should wait to update the description for the commit which starts using them.

Reply:
done, cruft cause I started with angular stagehand but later simplified